src/main/java/com/ifactory/press/db/solr/spelling/suggest/SafariInfixSuggester.java

```diff
@@ -28,7 +27,7 @@
 
     public SafariInfixSuggester(Version matchVersion, Directory dir, Analyzer indexAnalyzer, Analyzer queryAnalyzer,
             int minPrefixChars, boolean highlight) throws IOException {
-        super(matchVersion, dir, indexAnalyzer, queryAnalyzer, minPrefixChars);
+        super(dir, indexAnalyzer, queryAnalyzer, minPrefixChars, true);  // rivey - (matchVersion, take out Needs Testing
```

Comment:
Curious about this, did the parent's constructor signature change? Is `matchVersion` not needed anymore?

Reply:
It may be introspecting to get this. I couldn't find anywhere where it was needed in the code. This goes for future versions as well.